bmwqemu.pm

```diff
@@ -215,11 +215,13 @@ sub log_call {
     else {
         # key/value pairs
         my @result;
+        my $should_hide = 1 if grep { ($_ // '') eq 'secret' } @_;
```

Comment:
Declaring a variable with a nested `if` is a bit weird in my opinion. I'd write it simply as:
```suggestion
        my $should_hide = grep { ($_ // '') eq 'secret' } @_;
```